plugins/couchbase/couchbaseembed.cpp

```diff
@@ -912,32 +916,33 @@ namespace couchbaseembed
 
     IPropertyTree * CouchbaseEmbedFunctionContext::nextResultRowTree()
     {
-        for (auto cbrow : *m_pQuery)
+        if (m_pQuery)
         {
-            auto json = cbrow.json().to_string();
+            reportIfQueryFailure(m_pQuery);
```

Comment:
this makes sense. Per our offline conv, I'm wondering if reportIfQueryFailure should fail if m_pQuery contains multiple queries?
I'm not sure how often we'll encounter that situation, but I'm concerned about silently dropping the other queries.

Reply:
Well, with the original code, we never supported multiple queries anyway.  If we change anything, I would suggest only documenting this as a known limitation and then address it if the need arises.

Reply:
Sounds reasonable